### ISSUE TITLE
./toolchain.py pip install ignoring modules installed in conda environment

### ISSUE THREAD

**Issue by ssolari** [kivy/kivy-ios#347]
Using a base conda environment to build kivy-ios.   If a module already exists it is ignored by pip.

https://github.com/kivy/kivy-ios/blob/8d5c8c243f0eebf3649c216db34f6b097bdd05a9/toolchain.py#L1420

Fix for us was adding ```--ignore-installed``` above  as
```
pip_args = ["--isolated", "--ignore-installed", "--prefix", ctx.python_prefix] 
```

**Comment by hackalog** [comment]
Nice catch. I have exactly that diff in my tree, and had totally forgotten about why I had put it there.

**Comment by tito** [comment]
Fixed via #349
